components/brave_vpn/resources/panel/components/select-region-list/style.ts

```diff
@@ -95,18 +95,23 @@ export const RegionContainer =
 
   ${(p) => !p.selected &&
     css`
-      &:focus-within leo-button {
-        visibility: visible;
+      &:focus-visible leo-button {
```

Comment:
I think (but I'm not sure) if you change the selector to 

```suggestion
      &:has(:focus-visible) leo-button {
```

you might be able to get rid of the `showOnFocus` styles for `RegionConnect`

Reply:
Done with `&:has(:focus) leo-button {}`. `&:has(:focus-visible) leo-button {}` seems not working.